utils/swift_build_support/swift_build_support/products/product.py

```diff
@@ -191,24 +191,19 @@ def is_release(self):
         """
         return is_release_variant(self.args.build_variant)
 
-    def install_toolchain_path(self, host_target, force_local=False):
+    def install_toolchain_path(self, host_target):
         """toolchain_path() -> string
 
-        Returns the path to the local toolchain that is being created as part of
-        this build, or to a native prebuilt toolchain that was passed in.
+        Returns the path to the toolchain that is being created as part of this
+        build
         """
-        if self.args.native_swift_tools_path is not None and not force_local:
-            return os.path.split(self.args.native_swift_tools_path)[0]
```

Comment:
[I added this last year](https://github.com/apple/swift/pull/34437/files#diff-b84b1e74590d3a1234977cf25386482b020e325198c52f47625938c940a49e89R172) but I think it's better to do it explicitly where it's used, as shown below, plus I don't need this `force_local` nonsense.